src/data-display/dynamic/dynamic-layout/templates/markdown/index.vue

```diff
@@ -15,7 +15,7 @@ import {
     computed, getCurrentInstance,
     reactive, toRefs,
 } from 'vue';
-import type { Vue } from 'vue/types/vue';
+import type Vue from 'vue';
```

Comment:
Most of codes are working by the previous code. 
Is it ok ?

Reply:
Yes. My IDE returns error if import from 'vue/types/vue'